trueskill_test.go

```diff
@@ -242,6 +242,30 @@ func TestTrueSkill_MatchQuality_HeadToHead(t *testing.T) {
 	}
 }
 
+func TestWinProbability(t *testing.T) {
+	ts, err := NewDefault(0)
+	if err != nil {
+		t.Error(err)
+	}
+
+	teamA := Players{ts.NewDefaultPlayer(), ts.NewDefaultPlayer()}
+	teamB := Players{ts.NewDefaultPlayer()}
+
+	winProbability := ts.WinProbability(teamA, teamB)
+
+	wantProbability := 0.9
+	if !mathextra.Float64AlmostEq(winProbability, wantProbability, 1e-1) {
+		t.Errorf("Probability == %.1f, want %.1f", winProbability, wantProbability)
+	}
+
+	teamB = append(teamB, ts.NewDefaultPlayer())
+	winProbability = ts.WinProbability(teamA, teamB)
+	wantProbability = 0.5
+	if !mathextra.Float64AlmostEq(winProbability, wantProbability, 1e-1) {
```

Comment:
Hmm, here `1e-1` is much too lax, it doesn't take int account decimals following, e.g. result could be `0.51` or `0.59`, test wouldn't care.